docs: Expand Docker Config Secrets and private registry (#139)
kubewarden/docs: docs/operator-manual/policy-servers/02-private-registry.md

---
sidebar_label: "Using private registries"
title: ""
---

# Configuring PolicyServers to use private registries

It is possible to configure PolicyServers to use credentials of private OCI
registries. This will allow those PolicyServers to download policies from
public and private registries.

> Once a PolicyServer is configured to access private registries, policies running

Reviewer: This sentence is very long. Could we break it down into two separate sentences, please, as below?

Author: True! applied the suggestions, thanks!

on it and using the defined SDKs and lower level host capabilities APIs will be

> able to access private registries too, as PolicyServers expose that

Reviewer: ```suggestion
able to access private registries too. This is because PolicyServers expose that
```

functionality through the defined policy SDKs and lower level host capability

> API. This is the case for example in policies that verify signatures of

Reviewer: nit: Need to add comma
```suggestion
API. This is the case, for example, in policies that verify signatures of
```

container images.

To achieve this, we will create a Secret containing the private registry

> credentials, and configure our PolicyServers resources, and/or our Helm chart

Reviewer: nit
```suggestion
credentials, and configure our PolicyServers' resources, and/or our Helm chart
```

title: ""
---

# Configuring PolicyServers to use private registry

It is possible to configure PolicyServers to use credentials of private OCI registries.
This will allow those PolicyServers to download policies from public and private
registries.

> PolicyServers expose some functionality to pull OCI artifacts via the defined

Reviewer 2: ```suggestion
PolicyServers exposes some functionality to pull OCI artifacts via the defined
```
